### DISCUSSION TITLE
Missing releases folder?

### DISCUSSION THREAD

**Discussion by carbonga** [quarto-dev/quarto-cli#4146] [Q&A] [answered]
Hi there!
Bloody newbie here. I've been using Quarto to produce Rmarkdown-documents in RStudio, so thanks to the team are in order first! Now, I have run into an issue updating Quarto via Pamac, however. Pamac tries to update the newest version of Quarto-Cli from https://github.com/quarto-dev/quarto-cli/releases/download/v1.2.329/quarto-1.2.329-linux-amd64.deb but it seems that the releases folder has been (re)moved. The update process runs into a 404 error. Is there a problem with the file structure here on github or am I doing something wrong? Likely the latter. Any pointers would be greatly appreciated.
Kind regards!

**Comment by dragonstyle** [comment, marked as answer]
We ended up rolling back the release was had (so an earlier release is currently the latest). Perhaps this 'removal' of a previously published release has a downstream effect that we didn't understand / anticipate and this is the cause... We are planning to publish an updated 1.2 release later today, so that may resolve the issue.

**Reply by carbonga** [reply]
Thank you for your response! I'll try updating again tomorrow.

**Reply by carbonga** [reply]
Thank you for solving the problem re updating - I could update the package today without any issues. Hope not to have caused any issues regarding the original plan... Thanks in any case! :)
